Mage.Sets/src/mage/cards/r/RetributionOfTheAncients.java

```diff
@@ -36,7 +37,7 @@ public RetributionOfTheAncients(UUID ownerId, CardSetInfo setInfo) {
         // {B}, Remove X +1/+1 counters from among creatures you control: Target creature gets -X/-X until end of turn.
         DynamicValue xValue = new SignInversionDynamicValue(GetXValue.instance);
         Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new BoostTargetEffect(xValue, xValue, Duration.EndOfTurn, true), new ManaCostsImpl("{B}"));
-        ability.addCost(new RemoveVariableCountersTargetCost(filter, CounterType.P1P1, "X", 0));
+        ability.addCost(new RemoveVariableCountersTargetCost(StaticFilters.FILTER_CONTROLLED_CREATURES, CounterType.P1P1, "X", 0));
```

Comment:
You've replaced the filter here but you didn't remove the private filter definition above.

Reply:
done